phpBB/phpbb/di/container_builder.php

```diff
@@ -587,8 +593,10 @@ protected function inject_dbal_driver()
 					false,
 					defined('PHPBB_DB_NEW_LINK') && PHPBB_DB_NEW_LINK
 				);
+				$this->dbal_connection_doctrine = connection_factory::get_connection($this->config_php_file);
 			}
 			$this->container->set('dbal.conn.driver', $this->dbal_connection);
+			$this->container->set('dbal.conn.doctrine', $this->dbal_connection_doctrine);
```

Comment:
Why do we need to inject the connection here?

Reply:
for the same reason that we inject the classic driver: so we don't open more than one connection (well two now) per request

Reply:
This is never used though.

Reply:
Should double check whether this is needed